homeassistant/components/netatmo/camera.py

```diff
@@ -176,8 +182,10 @@ async def async_camera_image(self):
             return await self._data.async_get_live_snapshot(camera_id=self._id)
         except (
             aiohttp.ClientPayloadError,
-            pyatmo.exceptions.ApiError,
             aiohttp.ContentTypeError,
+            aiohttp.ServerDisconnectedError,
+            aiohttp.ClientConnectorError,
```

Comment:
This doesn't looks like clean up 🙂
But I guess it should be ok.

Reply:
Oh, right, another bug fix.  :-)